apps/framework-docs/src/pages/moose/moose-cli.mdx

```diff
@@ -154,6 +154,19 @@ moose generate migration --url https://<remote-env> --token <api-token> --save
 - Omit `--save` to output to stdout without writing files
 - Requires `[features].olap = true`; execution of saved plan requires `[features].ddl_plan = true`
 
+### DB Pull (External Tables)
```

Comment:
the taxonomy of commands / sections doesn't make much sense on this page. i feel like "db pull" and "generate migration" should go under "planning and deployment" section, or maybe "core commands", or maybe theres a new section "schema and migration management" or something like that? build and prod probably belong in deployment section? i also don't understand what refresh does, and how its different from db pull?